### PR TITLE
Fix typo in [en] resource_group_access_admin

### PR DESCRIPTION
**Pull request by Warfork**
### What does it do?
Corrects a typo in core/lexicon/en/access.inc.php

### CODE REVIEW

**Comment by rthrash** [comment]
This is a 100% correct copy edit fix. Good catch!

**Comment by opengeek** [comment]
Thank you for your contribution! Before your pull request can be reviewed, please [sign the MODX Contributor License Agreement (CLA)](https://modx.com/community/contribute/cla/). This is to ensure MODX has your written permission to distribute projects containing your contributions under the appropriate license.

Please make sure the CLA has been signed for GitHub user(s): @Warfork 

Once you've signed, please reply with a comment letting us know so we can verify.

**Review by Ibochkarev — APPROVED** [review]

**Comment by Warfork** [comment]
> Thank you for your contribution! Before your pull request can be reviewed, please [sign the MODX Contributor License Agreement (CLA)](https://modx.com/community/contribute/cla/). This is to ensure MODX has your written permission to distribute projects containing your contributions under the appropriate license.
> 
> Please make sure the CLA has been signed for GitHub user(s): @Warfork
> 
> Once you've signed, please reply with a comment letting us know so we can verify.

Signed.

**Comment by smg6511** [comment]
@Warfork - First, it's great to see someone new contributing! Just a quick word of advice: Try to make all the changes/commits you'd foresee making before submitting the pull. Otherwise, reviewers who've already approved are going to need to come back to re-review ... and that _could_ get a little confusing.  Cheers ;-)

**Comment by smg6511** [comment]
@Warfork - Also, please only make changes to the English lexicon files. All other languages are handled via Crowdin based on the current English versions.

**Comment by Warfork** [comment]
> @Warfork - Also, please only make changes to the English lexicon files. All other languages are handled via Crowdin based on the current English versions.

I'm really sorry to make this mistake. I will keep this in mind to prevent it from happening in the future. Do you want me to back and revert the changes to non-English lexicon files?

**Comment by Warfork** [comment]
> @Warfork - First, it's great to see someone new contributing! Just a quick word of advice: Try to make all the changes/commits you'd foresee making before submitting the pull. Otherwise, reviewers who've already approved are going to need to come back to re-review ... and that _could_ get a little confusing. Cheers ;-)

Yeah this was an error on my part. You also probably want to squash everything as well. I figured there was a trust factor and a chance you might not want to accept all the corrections so I split them up.

**Comment by smg6511** [comment]
> so I split them up

That's definitely a solid strategy, you just want to push all those separate commits to _your_ fork and wait until you think you're all done with the task at hand before submitting your PRs in the future.

Again, thanks for getting involved ... and ... keep it coming!

**Comment by Warfork** [comment]
> > so I split them up
> 
> That's definitely a solid strategy, you just want to push all those separate commits to _your_ fork and wait until you think you're all done with the task at hand before submitting your PRs in the future.
> 
> Again, thanks for getting involved ... and ... keep it coming!

Should I revert the non-English lexicon changes and make another PR here? Then hopefully you can squash the commit.

**Comment by smg6511** [comment]
> Should I revert the non-English lexicon changes and make another PR here?

@opengeek - Jason, would this be your preference? (@Warfork - FYI, Jason's the one who generally handles the merges.)

**Comment by Mark-H** [comment]
Yes, non-English changes will need to be reverted.

**Comment by Warfork** [comment]
> Yes, non-English changes will need to be reverted.

~~Is this applicable for the files modified in setup/lang/ as well?~~

Edit: I took a look at crowdin and noticed the setup folder is included. Please disregard this comment.

**Comment by Warfork** [comment]
Finished. After you review and squash the commits all should be well. I'm terribly sorry for over complicating something that's probably 10 lines of code in the end. Learning that you make use of crowdin as opposed to transifex was insightful.

**Comment by opengeek** [comment]
> Signed.

Confirmed! Thank you for contributing!

**Review by smg6511 — APPROVED** [review]

**Comment by opengeek** [comment]
@Warfork — why was this closed?

**Comment by Warfork** [comment]
> @Warfork — why was this closed?

@opengeek Sorry about that.

**Comment by Warfork** [comment]
@opengeek 

```
$_lang['setting_extension_packages'] = 'Extension Packages';
$_lang['setting_extension_packages_desc'] = 'A JSON array of packages to load on MODX instantiation. In the format [{"packagename":{"path":"path/to/package"}},{"anotherpackagename":{"path":"path/to/otherpackage"}}]';
```
Is this suppose to be installation?

**Comment by Warfork** [comment]
@opengeek I'm not actively looking for typos, but looking over documentation to learn modx. I guess this is seemingly endless in terms of what I could find. If you want me to stop making PR's let me know. I didn't bother fixing any grammatical errors as I don't have time to contribute in that capacity. I hope this helps your team.

File: core/lexicon/en/setting.inc.php [hunk 1]
```diff
@@ -261,7 +261,7 @@
 $_lang['setting_ext_debug_desc'] = 'Whether or not to load ext-all-debug.js to help debug your ExtJS code.';
 
 $_lang['setting_extension_packages'] = 'Extension Packages';
-$_lang['setting_extension_packages_desc'] = 'A JSON array of packages to load on MODX instantiation. In the format [{"packagename":{"path":"path/to/package"}},{"anotherpackagename":{"path":"path/to/otherpackage"}}]';
+$_lang['setting_extension_packages_desc'] = 'A JSON array of packages to load on MODX installation. In the format [{"packagename":{"path":"path/to/package"}},{"anotherpackagename":{"path":"path/to/otherpackage"}}]';
```
Comment (smg6511) [inline core/lexicon/en/setting.inc.php]: The original “instantiation” is correct here.

Comment (Warfork) [reply core/lexicon/en/setting.inc.php]: > The original “instantiation” is correct here.

@smg6511 Oops. Sorry for that.

**Review by smg6511 — APPROVED** [review]

**Review by Ibochkarev — APPROVED** [review]

**Review by Mark-H — APPROVED** [review]

**Review by opengeek — APPROVED** [review]

**Pull request merged**